frontend/controller/actions/identity.js

```diff
@@ -228,7 +228,6 @@ export default (sbp('sbp/selectors/register', {
     } catch (e) {
       console.error('gi.actions/identity/login failed!', e)
       const humanErr = L('Failed to login: {reportError}', LError(e))
-      alert(humanErr)
```

Comment:
I think this should be added back. Seems related to changes to #1371

Reply:
@Silver-IT but I think we want the user to see the error in this case, because it's a serious problem that would need manual intervention to solve, and this way if anyone runs into it they'll be able to take a screenshot of the error and we'll know. Most normal users don't check the console.

Reply:
Yeah, that's reasonable. Then, let's add it again.